fix(client-presence): ISessionClient naming consistency (#22973)
microsoft/FluidFramework: .changeset/ninety-zoos-trade.md

---
"@fluid-experimental/presence": minor
---
---
"section": feature
---

> Presence `ISessionClient` connectivity status added

Reviewer: ```suggestion
ISessionClient now exposes connectivity information
```

> 1. `ISessionClient` is updated to indicate current connectivity status via new method `getConnectionStatus()` with two possible states `Connected` and `Disconnected`.
> 2. `ISessionClient` member `connectionId()` has been renamed to `getConnectionId()` for consistency.
> 3. `IPresence` event `attendeeDisconnected` is now implemented.

Reviewer: ```suggestion
1. `ISessionClient` has a new method, `getConnectionStatus()`, with two possible states: `Connected` and `Disconnected`.
2. `ISessionClient`'s `connectionId()` member has been renamed to `getConnectionId()` for consistency.
3. `IPresence` event `attendeeDisconnected` is now implemented.
```
